scripts/connectors.json

```diff
@@ -54,6 +54,27 @@
             "path": "polygon-bridge/main.sol",
             "slug": "assets-polygon-bridge",
             "title": "Assets Transfer Polygon Bridge"
+        },
+        {
+            "path": "INST/main.sol",
+            "slug": "inst",
+            "title": "INST"
+        },
+        {
+            "path": "instapool_v2/main.sol",
+            "slug": "instapool_v2",
+            "title": "Instapool V2"
```

Comment:
I can't find `Instapool V2` on the docs: https://docs-nuxt-git-staging-instadapp-eng.vercel.app/connectors/mainnet

Am I missing something?